src/app/components/annotations/components/annotation-search-form/annotation-search-form.component.ts

```diff
@@ -231,16 +277,24 @@ export class AnnotationSearchFormComponent implements OnInit {
       return;
     }
 
-    this.searchParameters.sort = event.target.value;
     this.searchParametersChange.emit(this.searchParameters);
   }
 
-  protected toNumber(stringValue: string): number | null {
-    const value = Number(stringValue);
-    if (isNaN(value)) {
-      return null;
+  // While I could add a function to the template to validate that the score is
+  // valid, that would not be very performant because we'd be checking if the
+  // score is valid even when unrelated properties are updated.
+  //
+  // TODO: Replace this with a computed signal once this component migrates to
+  // onPush + signal-based reactivity.
+  private validateScoreParameters() {
+    if (
+      this.searchParameters.scoreLowerBound !== null &&
+      this.searchParameters.scoreUpperBound !== null &&
+      this.searchParameters.scoreLowerBound > this.searchParameters.scoreUpperBound
+    ) {
+      this.scoreRangeError.set("Score lower bound must be less than or equal to the score upper bound.");
```

Comment:
```suggestion
      this.scoreRangeError.set("Score minimum must be less than or equal to the score maximum.");
```

Should use consistent language with the labels on the inputs